language/tools/move-mv-llvm-compiler/src/stackless/llvm.rs

```diff
@@ -491,11 +483,9 @@ impl Builder {
             assert_eq!(src.0 .0, stype.0);
             let srcval = LLVMBuildLoad2(self.0, stype.0, src.1 .0, "srcval".cstr());
 
-            // The LLVM struct currently has one additional compiler-generated field. We won't
-            // extract that for an unpack.
             let user_field_count = dst.len();
             assert_eq!(
-                user_field_count + 1,
+                user_field_count,
```

Comment:
I've never liked this ` + 1` :)

Reply:
Agreed, this is cleaner, and it was a bit of a pain keeping every place in sync (hence the copious assertions to make sure all places did agree).